### PR TITLE
Add tutorial to CHANGES.md to prepare for release v2.1.0

### PR DESCRIPTION
**Pull request by andreanne-lemay**
The last milestone for release v2.1.0 has been merged. This PR updates the CHANGES.md file

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -22,4 +23,4 @@
  - Optimize binarization.  [View pull request](https://github.com/ivadomed/ivadomed/pull/364)
  - Automate training improvement.  [View pull request](https://github.com/ivadomed/ivadomed/pull/362)
  - Complement to PR 342.  [View pull request](https://github.com/ivadomed/ivadomed/pull/361)
```
Comment (jcohenadad) [inline CHANGES.md:25]: can we update the title of this PR to be more explicit

Comment (andreanne-lemay) [reply CHANGES.md:25]: Done!

Comment (andreanne-lemay) [reply CHANGES.md:25]: Ah wrong PR! Im on it

Comment (andreanne-lemay) [reply CHANGES.md]: Ok done :)!

**Pull request merged**